packages/app/src/cli/models/app/loader.test.ts

```diff
@@ -2364,7 +2367,7 @@ wrong = "property"
       cmd_app_all_configs_any: true,
       cmd_app_all_configs_clients: JSON.stringify({'shopify.app.toml': '1234567890'}),
       cmd_app_linked_config_name: 'shopify.app.toml',
-      cmd_app_linked_config_git_tracked: false,
+      cmd_app_linked_config_git_tracked: true,
```

Comment:
I actually forced this to true to test that the new `checkIfGitTracked` works correctly

Previously it was false because `checkIfIgnoredInGitRepository` needs to load a valid git repository to work (which is slow) and was always false in this test.